cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,80 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="wrap-instructor-info studio-view maintenance-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
```

Comment:
But that form uses a fieldset with a legend. This one doesn't.  The sign up form adds the sr class to the legend to move it offscreen, which is fine.  Something similar here would work.  It would be helpful if the legend contained text that described what the form will do once submitted.